app/lib/travis_event.rb

```diff
@@ -11,7 +11,7 @@ class TravisEvent
   # the next steps. If not kick off a Sentry error.
 
   def initialize(payload)
-    Raven.capture_message('Unknown Travis Event Received', extra: payload)
+    Raven.capture_message('Unknown Travis Event Received', extra: payload) rescue nil
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using `rescue` in its modifier form.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5421819)